src/components/claim/ClaimRewardsVeDF.svelte

```diff
@@ -147,7 +147,7 @@
         : ""
     }`}
     apyTooltip={descriptions.default.tooltip_rewards_apy_active}
-    description="Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>."
+    description="Earn Active Rewards from Data Farming by <strong>allocating veOCEAN</strong> and <strong>curating quality data</strong>.<br/>Get <strong>2X Rewards by publishing</strong> your own datasets and allocating to them."
```

Comment:
Don't use a <br/>, the styling/formatting looks bad.

Can you also update the Passive Rewards card to not have a <br/>? It doesn't look good at the moment.

Reply:
can you also add some spaces between paragraphs. It creates some space to breath. 

<img width="576" alt="Screenshot 2023-02-21 at 11 36 51" src="https://user-images.githubusercontent.com/13741335/220390396-0783bc9b-7160-4fe7-b80f-44147a3ca5f1.png">

Reply:
@EnzoVezzaro sure, looks better